docs/dm/settings.md

```diff
@@ -77,6 +77,6 @@ As time went on however the triangle mode became the default as it is much more
 
 These settings are additional tuning factors to the vision system and determine limits to the vision of all tokens.
 
-Suppose there is one giant open field that stretches 5000 ft. and is completely illuminated. One could expect the entire area to be visible to your token standing in the middle of this field.  There are however limitations to the human eye, hence the introduction of these settings.
+Suppose there is one giant open field that stretches 5000 ft. and is completely illuminated. One could expect the entire area to be visible to your token standing in the middle of this field.  There are however limitations to the human (and the most non-human, as well) eye, hence the introduction of these settings.
```

Comment:
I don't mind the joke, but the sentence feels of, maybe 'and the' should be 'and to'